config.yaml

```
@@ -10,7 +11,7 @@ options:
       If unset, the manifests will use the image registry from the kube-control relation
 
       example)
-        juju config kubernetes-metrics-server image-registry='rocks.canonical.com:443/cdk'
+        juju config kubernetes-metrics-server image-registry='k8s.gcr.io'
```

is this supposed to be an example of an empty registry config? 
```suggestion
        juju config kubernetes-metrics-server image-registry=''
```

---

uh... yes you're right!

---

also, is the metrics-server a copy paste error?

---

yes, correct.  Let me fix that too